### PR TITLE
Test for parseError using WITH and UNION ALL operators

### PR DESCRIPTION
**Pull request by Atri0**

### CODE REVIEW

**Comment by Atri0** [comment]
@Agirish please review

File: framework/resources/Functional/union_all/prq_union_all/prqUAllprq_36.sql [hunk 1]
```diff
@@ -0,0 +1,9 @@
+drop table if exists dfs.union_all.`MyTbl_1`;
+drop table if exists dfs.union_all.`MyTbl_2`;
+drop view if exists dfs.union_all.`MyView1`;
+create table dfs.union_all.MyTbl_1 as (select 1 as Col);
+create table dfs.union_all.MyTbl_2 as (select 2 as Col);
+use dfs.tmp;
```
Comment (Agirish) [inline framework/resources/Functional/union_all/prq_union_all/prqUAllprq_36.sql]: I'm a bit confused. We're using dfs.tmp here and dfs.union_all above and below

**Comment by Agirish** [comment]
One minor clarification needed. Can merge after that

File: framework/resources/Functional/union_all/prq_union_all/prqUAllprq_36.sql (hunk 1 above)
Comment (denysord88) [reply framework/resources/Functional/union_all/prq_union_all/prqUAllprq_36.sql]: Fixed.

**Comment by denysord88** [comment]
Fixed the query workspace. @Agirish could you please review the changes?

**Pull request merged**